CHANGELOG.md

```diff
@@ -42,12 +46,19 @@ be found.
 * LCOW: Remove hard-coding [moby/moby#34398](https://github.com/moby/moby/pull/34398)
 * LCOW: WORKDIR correct handling [moby/moby#34405](https://github.com/moby/moby/pull/34405)
 * Windows: named pipe mounts [moby/moby#33852](https://github.com/moby/moby/pull/33852)
+* Fix layers size reported as 0 in `docker system df` [moby/moby#34826](https://github.com/moby/moby/pull/34826)
+* Fix `device or resource busy` bug when removing stopped container [moby/moby#34886](https://github.com/moby/moby/pull/34886)
```

Comment:
I slightly changed it in my patch to "some" device or resource busy issues, because this is just _one_ possible situation (there may still be others that are not addressed)

Reply:
i'll remove this line

Reply:
No it's in Runtime (but the diff skipped those lines :D)